docs/content/v0.20/examples.md

```diff
@@ -59,29 +59,29 @@ manager which connects to the `wds1` front-end and the `imbs1` OCM control plane
 
 6. Follow the steps to [create and register two clusters with OCM](example-wecs.md).
 
-7. (optional) Check all deployments and statefulsets running in the hosting cluster. Expect to
-see the wds1 kubestellar-controller-manager created in the wds1-system namespace and the imbs1
-statefulset created in the imbs1-system namespace.
+7. (optional) Check relevant deployments and statefulsets running in the hosting cluster. Expect to
+see the `kubestellar-controller-manager` in the `wds1-system` namespace and the 
+statefulset `vcluster` in the `imbs1-system` namespace, both in a `READY` state.
```

Comment:
The wording here is off. It implies that tables have a STATE column that might or might not hold a value "READY". I would reword, something like

> ... both fully ready

or

> ... both fully ready (number of available replicas equals number of desired replicas).

Reply:
fixed